update-docs.ps1

```diff
@@ -22,18 +22,31 @@ param (
 . ./shared.ps1
 
 function Update-Canonical-Data {
-    Write-Output "Updating canonical data"
-    Run-Command "./update-canonical-data.ps1" 
+    [CmdletBinding(SupportsShouldProcess)]
+    param ()
+
+    if ($PSCmdlet.ShouldProcess($true)) {
+        Write-Output "Updating canonical data"
+        Invoke-CommandExecution "./update-canonical-data.ps1" 
+    }
 }
 
-function Update-Docs {
-    Write-Output "Updating docs"
-    $args = if ($Exercise) { @("-o", $Exercise) } else { @() }
-    Run-Command "./bin/fetch-configlet"
-    Run-Command "./bin/configlet generate . -p problem-specifications $args"
+function Update-DocFile {
```

Comment:
Why this naming change?

Reply:
PSAnalyzer rejects plurals:
PSUseSingularNouns

Reply:
Ah, I find that a weird rule, but let's see if we can work around that. I don't like `Update-DocFile` because it suggests only one doc file is changed and that is just incorrect. In general, if the singular version is not actually correct, I'd prefer disabling the analyzer rule over a valid name that is incorrect.

That said, would `Update-Documentation` work here?

Reply:
Yes, or Update-DocsFolder ?

I think this is what actually happens, some files in the folder docs are updated ?

Reply:
Yeah that could work